pyproject.toml

```diff
@@ -7,6 +7,7 @@ license = { file = "LICENSE.md", content-type = "text/markdown" }
 version = "0.1.0"
 dependencies = [
   "evo-data-converters-common",
+  "evo-data-converters-duf",
```

Comment:
As a Mac user, how is this going to affect me when I am maintaining the monorepo?

Reply:
Long term we are expecting deswik to make their libraries work on Linux, which provides more flexibility

Reply:
Ah yes, that would be a good solution for those working in the monorepo. I don't think Python has a way to specify that packages cannot be installed on specific platforms, other than runtime errors.